Dfe.PrepareTransfers.Data.TRAMS.Tests/Mappers/Response/TramsTrustMapperTests.cs

```diff
@@ -21,86 +23,81 @@ public TramsTrustMapperTests()
         [Fact]
         public void GivenTramsTrust_ItMapsTrustFieldsCorrectly()
         {
-            var trustToMap = new TramsTrust
+            var trustToMap = new TrustDto
             {
-                GiasData = new TramsTrustGiasData
-                {
+
                     CompaniesHouseNumber = "1231231",
-                    GroupContactAddress = new GroupContactAddress
+                    Address = new AddressDto
                     {
-                        AdditionalLine = "Extra line",
+                        Additional = "Extra line",
                         County = "County",
                         Locality = "Locality",
                         Postcode = "Postcode",
                         Street = "Street",
                         Town = "Town"
                     },
-                    GroupId = "0001",
-                    GroupName = "Trust name",
+                    ReferenceNumber = "0001",
+                    Name = "Trust name",
                     Ukprn = "1001",
-                },
-                IfdData = new TramsTrustIfdData()
-                {
-                    LeadRscRegion = "London"
-                }
             };
 
             var result = _subject.Map(trustToMap);
             var expectedAddress = new List<string> {"Trust name", "Street", "Town", "County, Postcode"};
 
-            Assert.Equal(trustToMap.GiasData.CompaniesHouseNumber, result.CompaniesHouseNumber);
-            Assert.Equal(trustToMap.GiasData.GroupId, result.GiasGroupId);
-            Assert.Equal(trustToMap.GiasData.GroupName, result.Name);
-            Assert.Equal(trustToMap.GiasData.Ukprn, result.Ukprn);
+            Assert.Equal(trustToMap.CompaniesHouseNumber, result.CompaniesHouseNumber);
+            Assert.Equal(trustToMap.ReferenceNumber, result.GiasGroupId);
+            Assert.Equal(trustToMap.Name, result.Name);
+            Assert.Equal(trustToMap.Ukprn, result.Ukprn);
             Assert.Equal("Not available", result.EstablishmentType);
             Assert.Equal(expectedAddress, result.Address);
-            Assert.Equal(trustToMap.IfdData.LeadRscRegion, result.LeadRscRegion);
+            //Assert.Equal(trustToMap.IfdData.LeadRscRegion, result.LeadRscRegion);
         }
 
-        [Fact]
-        public void GivenTrustWithOneAcademy_MapASingleAcademy()
-        {
-            _establishmentMapper.Setup(m => m.Map(It.IsAny<TramsEstablishment>()))
-                .Returns<TramsEstablishment>(input => new Academy {Ukprn = $"Mapped {input.Ukprn}"});
+        //ToDo: fix these once we can get establishments by ukprn
```

Comment:
![image](https://github.com/DFE-Digital/prepare-academy-transfers/assets/113213942/2518a5e6-8ef7-4ee8-b689-6277562642cb)
We have this, but is what you're after multiple establishments with the same ukprn?

Reply:
Oh or is it by trust ukprn
that's now in I think 
![image](https://github.com/DFE-Digital/prepare-academy-transfers/assets/113213942/81d757b6-5b6b-4500-85d2-c02acb05ae5c)